google-cloud-bigtable/lib/google/cloud/bigtable/admin/v2/bigtable_instance_admin_client.rb

```diff
@@ -136,6 +136,10 @@ def self.location_path project, location
               )
             end
 
+            class OperationsClient < Google::Longrunning::OperationsClient
+              SERVICE_ADDRESS = SERVICE_ADDRESS
```

Comment:
Is this correct?

Reply:
Yes, it's setting `OperationsClient::SERVICE_ADDRESS` to `BigtableInstanceAdminClient::SERVICE_ADDRESS`. (We could probably improve the code generator to be explicit here, since this is confusing...)

Reply:
Wow, how does this work?

Reply:
It works. But I agree it looks confusing.

```ruby
class Foo
  VERSION = "1.0"
end

class Bar < Foo
  VERSION = VERSION
end

class Baz < Foo
  # Don't explicitly set the constant
end

Foo::VERSION #=> "1.0"
Bar::VERSION #=> "1.0"
Baz::VERSION #=> "1.0"

# Change Foo's constant
Foo::VERSION.gsub! "1.0", "1.1"

Foo::VERSION #=> "1.1"
Bar::VERSION #=> "1.0"
Baz::VERSION #=> "1.1"
```

Reply:
Filed https://github.com/googleapis/toolkit/issues/1751.